### ISSUE TITLE
One form to rule them all

### ISSUE THREAD

**Issue by ScottDowne** [mozilla/donate.mozilla.org#199]
Right now we submit with multiple forms. One for stripe, one for paypal single donation, and one for paypal recurring.

Our new single page form which breaks things into components needs a way to post to the server.

We should update this to be one form and have the sever side respond to the radio button value and if it's paypal or stripe.

**Comment by adamlofting** [comment]
:+1:

**Comment by adamlofting** [comment]
@ScottDowne Are we doing this as part of the existing work this week?

If not, and it's a new task, please update to Aug 28 Milestone.

**Comment by ScottDowne** [comment]
I already have this work in another pull request. It needs to be rebased again and is blocked on another pull request. So it shouldn't be a lot more work, but won't be landing right away.

**Comment by ScottDowne** [comment]
This ends my refactoring for the week. https://github.com/mozilla/donate.mozilla.org/commit/567709a0b1a7b9f7d43e2bd37084bbff204889a4
